cadquery/occ_impl/shapes.py

```diff
@@ -2517,7 +2557,7 @@ def fillet(
     ) -> "Wire":
         """
         Apply 2D or 3D fillet to a wire
-
+        :param wire: The input wire to fillet. Currently only open wires are supported
```

Comment:
This docstring line was removed in #1630.   Probably got inadvertently added back.

Reply:
Thanks, fixed!